tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -1875,7 +1822,7 @@ namespace OutputProcessor {
                                           StoreType::Averaged,
                                           1,
                                           -999,
-                                          "indexGroup",
+                                          OutputProcessor::eTimeStepType::Zone,
```

Comment:
Passing the new enum into `WriteReportVariableDictionaryItem` instead of the string `"indexGroup"`.  In the code, this function is called from SetupOutputVariable, and the TimeStepType argument is forwarded to this function.  It goes through a validation function that verifies it matches one of the expected time step types (Zone, HVAC, System, etc.).  It would throw a fatal if you tried to call SetupOutputVariable with an argument of "indexGroup".  So I don't quite get why this unit test is passing "indexGroup" in.  I modified it to pass the enum type and it is good to go.